tests/static/test-base-ssh.json

```diff
@@ -0,0 +1,5 @@
+{
+    "ifconfig": "br-lan    Link encap:Ethernet  HWaddr 08:00:27:EA:D5:D7  \n          inet addr:192.168.56.2  Bcast:192.168.56.255  Mask:255.255.255.0\n          inet6 addr: fd13:3fc2:e81f::1/60 Scope:Global\n          inet6 addr: fe80::a00:27ff:feea:d5d7/64 Scope:Link\n          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1\n          RX packets:50 errors:0 dropped:0 overruns:0 frame:0\n          TX packets:43 errors:0 dropped:0 overruns:0 carrier:0\n          collisions:0 txqueuelen:1000 \n          RX bytes:7379 (7.2 KiB)  TX bytes:6912 (6.7 KiB)\n\neth0      Link encap:Ethernet  HWaddr 08:00:27:EA:D5:D7  \n          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1\n          RX packets:51 errors:0 dropped:0 overruns:0 frame:0\n          TX packets:43 errors:0 dropped:0 overruns:0 carrier:0\n          collisions:0 txqueuelen:1000 \n          RX bytes:8145 (7.9 KiB)  TX bytes:6930 (6.7 KiB)\n\neth1      Link encap:Ethernet  HWaddr 08:00:27:D9:10:14  \n          inet addr:10.0.2.4  Bcast:10.0.2.255  Mask:255.255.255.0\n          inet6 addr: fe80::a00:27ff:fed9:1014/64 Scope:Link\n          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1\n          RX packets:19 errors:0 dropped:0 overruns:0 frame:0\n          TX packets:31 errors:0 dropped:0 overruns:0 carrier:0\n          collisions:0 txqueuelen:1000 \n          RX bytes:2984 (2.9 KiB)  TX bytes:3524 (3.4 KiB)\n\nlo        Link encap:Local Loopback  \n          inet addr:127.0.0.1  Mask:255.0.0.0\n          inet6 addr: ::1/128 Scope:Host\n          UP LOOPBACK RUNNING  MTU:65536  Metric:1\n          RX packets:20 errors:0 dropped:0 overruns:0 frame:0\n          TX packets:20 errors:0 dropped:0 overruns:0 carrier:0\n          collisions:0 txqueuelen:1000 \n          RX bytes:1728 (1.6 KiB)  TX bytes:1728 (1.6 KiB)",
+    "olsrd -v": "*** olsr.org - 0.9.7-git_0000000-hash_0e6c6b8511b8405eee05046434d38c1b ***\n http://www.olsr.org\n\nTerminated",
+    "iwconfig": "lo        no wireless extensions.\n\nenp1s0    no wireless extensions.\n\nwlp2s0    IEEE 802.11  ESSID:\"MY ROUTER\"  \n          Mode:Managed  Frequency:2.457 GHz  Access Point: 83:A8:1C:62:A3:56   \n          Bit Rate=150 Mb/s   Tx-Power=20 dBm   \n          Retry short limit:7   RTS thr:off   Fragment thr:off\n          Power Management:off\n          Link Quality=62/70  Signal level=-48 dBm  \n          Rx invalid nwid:0  Rx invalid crypt:0  Rx invalid frag:0\n          Tx excessive retries:1  Invalid misc:348   Missed beacon:0\n"
+}
```

Comment:
Can you format this so it's readable please?
Also ensure there's the end line at the end of the file please.

Reply:
> Can you format this so it's readable please?

How? I can not split it into multiple lines because this is JSON, is there anything else I can do?

Reply:
Other than JSON, we could use CSV but I don't know how much better it would be than this. Or we could just save these outputs in their own separate `.txt` files, but with the SNMP tests we are going to have too many files for the number of OIDs there are. What do you think?

Reply:
Because of limitations of JSON, the only way I could think of to maintain readability is to wrap every line in quotes and save the list of lines as value (Done in https://github.com/openwisp/netengine/pull/48/commits/3c20fd81375bfe8d58304986e0e44c46bece7f40). Please let me know if this is good enough. Other than that, I can't think of a good way to store a result, we could for example store a `.txt` file for each command, but with the SNMP tests we will have too many files (one for each OID)